source/hardware/controllers/mvave_smc_mixer.rst

```diff
@@ -0,0 +1,116 @@
+M-Vave SMC-Mixer
+================
+
+`Manufacturer's product page <https://www.cuvave.com/productinfo/1106102.html>`_ ·
+`Forum thread <https://mixxx.discourse.group/t/m-wave-sinco-smc-mixer-radio-broadcast-mapping/30366>`_
+
+.. versionadded:: 2.5.1
+
+Overview
+--------
+
+.. figure:: ../../_static/controllers/mvave_smc_mixer.svg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: M-Vave SMC-Mixer
+   :figclass: pretty-figures
+
+See the tables below for controls descriptions.
+
+.. note:: The mixer can be connected over Bluetooth, USB-C, or both. For use with Mixxx it is recommended to disable Bluetooth and connect only via USB-C.
+
+Deck and EQ
+-----------
+
+On the left side of the controller each set of a fader, rotary encoder, and
+buttons affect a single deck starting with 3 on the left, then 1, 2, and finally
+deck 4.
+
+..
+
+.. csv-table::
+   :header: "Control", "Name", "Function"
+   :widths: 5 25 70
+
+   "Knob", "(rotate)", "Adjust selected function (defaults to channel gain)"
+   "Treble", ":hwlabel:`M` (press)", "Toggle high filter cutoff"
+   "Treble", ":hwlabel:`M` (long press)", "Toggle knob between gain and high filter"
+   "Mid", ":hwlabel:`S` (press)", "Toggle mid filter cutoff"
+   "Mid", ":hwlabel:`S` (long press)", "Toggle knob between gain and mid filter"
+   "Bass", ":hwlabel:`R` (press)", "Toggle low filter cutoff"
+   "Bass", ":hwlabel:`R` (long press)", "Toggle knob between gain and low filter"
+   "Quick Effect", ":hwlabel:`□` (press)", "Toggle quick effect rack"
+   "Quick Effect", ":hwlabel:`□` (long press)", "Toggle knob between gain and quick effect"
+   "LEDs", "", "Blinks when the hardware control does not match the software control and soft takeover is enabled"
+   "Faders", "", "Adjust volume level"
```

Comment:
I don't want mess with your mapping, but some of this doesn't seem particularly useful to me. Do you expect this controller to be used with other software? How realistic is it that the controls diverge? Maybe some other status would be more useful, for example whether the deck is playing or not?

Reply:
This is the original function of these LEDs, and I've found it very useful on other controllers that have this feature (the Traktor S4Mk3, for example). This is something we could put behind a setting later though in a backwards compatible way, so maybe we can leave it alone for now and think about it more later to avoid having to do yet another round of changing stuff on the original mapping?

Reply:
Okay, in that case, the defaults have priority. Thank you for the explanation.